### ISSUE TITLE
import component with error of window is not defined

### ISSUE THREAD

**Issue by imink** [fullstackreact/google-maps-react#38]
I am going to import the the component which cause the following errors:

```
[2] [dev:server]
[2] [dev:server] /Users/imink/workspace/AlwaysAround/aa_admin/node_modules/google-maps-react/dist/lib/ScriptCache.js:5
[2]         factory(exports);
[2]         ^
[2] [dev:server] ReferenceError: window is not defined
[2]     at /Users/imink/workspace/AlwaysAround/aa_admin/node_modules/google-maps-react/dist/lib/ScriptCache.js:20:9
[2]     at Object.defineProperty.value (/Users/imink/workspace/AlwaysAround/aa_admin/node_modules/google-maps-react/dist/lib/ScriptCache.js:5:9)
[2]     at Object.<anonymous> (/Users/imink/workspace/AlwaysAround/aa_admin/node_modules/google-maps-react/dist/lib/ScriptCache.js:13:3)
[2]     at Module._compile (module.js:570:32)
[2]     at Object.Module._extensions..js (module.js:579:10)
[2]     at Module.load (module.js:487:32)
[2]     at tryModuleLoad (module.js:446:12)
[2]     at Function.Module._load (module.js:438:3)
[2]     at Module.require (module.js:497:17)
[2]     at require (internal/module.js:20:19)
```

My custom component is:
```
import React, { PropTypes as T } from 'react'
import Map, {GoogleApiWrapper, Marker} from 'google-maps-react'


let moved = false;

export class MapComp extends React.Component {


  render() {
    if (!this.props.loaded) {
      return <div>Loading...</div>
    }
    return (
      <div>Map will go here</div>
    )
  }
}

export default GoogleApiWrapper({
  apiKey: "my_own_key"
})(MapComp)
```

Any thoughts?

**Comment by diego-betto** [comment]
I think that is because there is no "window" in server side rendering

**Comment by auser** [comment]
Merged a new version that will fix this (`v1.1.0`)

**Comment by himanshuteotia** [comment]
It is still not working I have just installed the v1.1.0

**Comment by mawiza** [comment]
I have the same issue. Installed v1.1.0

**Comment by Eschon** [comment]
It looks like the fix is not actually in v1.1.0
If you look at the projects commit history the fix happend after v1.1.0
